RevenueCatUI/CustomerCenter/URLUtilities.swift

```diff
@@ -74,6 +74,19 @@ enum URLUtilities {
 
 }
 
+extension URL {
+
+    var isDeeplink: Bool {
+        switch scheme?.lowercased() {
+        case "http", "https":
+            return false
```

Comment:
On Android, clicking e.g. a Twitter link will deeplink into the Twitter app. That's an `https` URL. Is something like that possible on iOS? If so, we can't assume it's not a deeplink based on the scheme.

Reply:
the main issue is that `self.inAppBrowserURL = .init(url: url)` crashes if it's not `http` or `https`. But you have a point so maybe `isDeeplink` is not that accurate. I do think `isDeeplink` is easy to understand what it does tho